lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/expression/ConnectorExpressionPatterns.java

```diff
@@ -90,6 +96,11 @@ private ConnectorExpressionPatterns() {}
         });
     }
 
+    public static Property<Call, ?, List<ConnectorExpression>> arguments()
```

Comment:
Doesn't belong to "Translate IN predicate to connector expression" commit

Reply:
Actually unused. Removed.